sa/sa.go

```diff
@@ -636,12 +636,33 @@ func (ssa *SQLStorageAuthority) UpdateRegistration(ctx context.Context, reg core
 // NewPendingAuthorization stores a new Pending Authorization
```

Comment:
Can you update this comment to mention that there may not be a brand new pending authorization created depending on whether a reuse occurs?